### PR TITLE
lesson 2 finish

### PR DESCRIPTION
**Pull request by Alex-javac**

### CODE REVIEW

File: .idea/shelf/Uncommitted_changes_before_Checkout_at_25_02_2021_00_27_[Default_Changelist]/shelved.patch [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Index: .idea/workspace.xml
```
Comment (TheFallen058) [inline .idea/shelf/Uncommitted_changes_before_Checkout_at_25_02_2021_00_27_[Default_Changelist]/shelved.patch:1]: it's better to remove this files from git at all

File: src/lesson2/Task6.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package lesson2;
+
+import java.util.Random;
+
+//Создайте число. Определите, является ли число трехзначным. Определите, является
```
Comment (TheFallen058) [inline src/lesson2/Task6.java:5]: Please use 'Reformat code option'

File: src/lesson2/Task13.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package lesson2;
+
+import java.util.Random;
+
+//Создать последовательность случайных чисел, найти и вывести наибольшее из них.
+public class Task13 {
```
Comment (TheFallen058) [inline src/lesson2/Task13.java:6]: of course I expect to see initial array)

File: src/lesson2/Task13.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[] arr =new int[10];
+for (int i = 0;i < arr.length;i++){
+arr[i]=random.nextInt(1000);
+}
+int max=arr[0];
+for(int i=0;i<arr.length;i++){
```
Comment (TheFallen058) [inline src/lesson2/Task13.java:14]: could be just enhanced for
for (int number : arr) {

File: src/lesson2/Task14.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        int[] arr =new int[random.nextInt(20)];
+        for (int i = 0;i < arr.length;i++){
+            arr[i]=random.nextInt(10);
+            System.out.println("индекс "+i+" оценка "+arr[i]);
+        }
+        int min=arr[0];
+        int indexMin=0;
+        int max=arr[0];
+        int indexMax=0;
+        for(int i=0;i<arr.length;i++){
```
Comment (TheFallen058) [inline src/lesson2/Task14.java:19]: one loop - great solution!

**Review by TheFallen058 — APPROVED** [review]
You did this task very well!
some small notes from my side
good variable names in general, good result description

**Comment by Alex-javac** [comment]
thanks for your notes, it's very useful for me!

File: .idea/shelf/Uncommitted_changes_before_Checkout_at_25_02_2021_00_27_[Default_Changelist]/shelved.patch (hunk 1 above)
Comment (Alex-javac) [reply .idea/shelf/Uncommitted_changes_before_Checkout_at_25_02_2021_00_27_[Default_Changelist]/shelved.patch:1]: it's just add in gitignor (*xml, *iml) ?

Comment (Alex-javac) [reply .idea/shelf/Uncommitted_changes_before_Checkout_at_25_02_2021_00_27_[Default_Changelist]/shelved.patch:1]: Is it just add in gitignor (*xml, *iml) ?

**Pull request merged**